lib/build.api.js

```diff
@@ -136,23 +153,27 @@ class Build {
           if (key !== 'root' && key !== 'contrib') {
             links.push({
               src: project.config.paths.modules[key],
-              dest: 'sites/all/modules/' + key,
+              dest: path.join(versionDestPaths.modules, key),
               type: 'dir'
             });
           }
         });
 
-        // Add custom themes link.
-        links.push({
-          src: project.config.paths.themes.custom,
-          dest: 'sites/all/themes/custom',
-          type: 'dir'
+        // Add themes links.
+        Object.keys(project.config.paths.themes).forEach((key) => {
```

Comment:
What this does though is kinda simplify the "map" above. Notice that the themes directory is the only one that specifically calls for "custom". That being said, it was just "wierd" to me to have a map that was "themesCustom"